jupytercalpoly/Jupyter-Bifrost
# Fix #42

## Changes
* Fixed #42 by saving types of columns on the backend.
* Fixed the given encoding to the suggested spec so that chart chooser only suggests the graph with the provided columns on the right axis.
* A few changes on select columns:
    1. Changed to use compass QL when the number of selected columns are less than four.
    2. If the number of selected columns is bigger than three, added the permutation of non-fixed columns to the fixed columns so that the size of the encodings becomes three.

## Testing
1. Plot bifrost without any parameters or with parameters
2. See the changes in chart chooser
3. See if you can add a new variable pill in data tab.
